api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/MessagePayloadFactory.java

```diff
@@ -58,17 +61,37 @@ public static EmailMessagePayload buildSendNewCardSuccessEmailPayload(PaymentCyc
     }
 
     public static EmailMessagePayload buildPaymentNotificationEmailPayload(PaymentCycle paymentCycle) {
-        Map<String, Object> emailPersonalisation = createPaymentEmailPersonalisationMap(paymentCycle);
+        Map<String, Object> emailPersonalisation = createPaymentEmailPersonalisationMap(paymentCycle, paymentCycle.getVoucherEntitlement());
         return EmailMessagePayload.builder()
                 .claimId(paymentCycle.getClaim().getId())
                 .emailType(EmailType.PAYMENT)
                 .emailPersonalisation(emailPersonalisation)
                 .build();
     }
 
-    private static Map<String, Object> createPaymentEmailPersonalisationMap(PaymentCycle paymentCycle) {
+    /**
+     * Builds the message payload for the email that gets sent should one of the claimants children be turning
+     * 4 during the next payment cycle.
+     *
+     * @param paymentCycle                       The current payment cycle
+     * @param entitlementNextMonth               The entitlement that the claimant will be receiving next month
+     * @param multipleChildrenTurningFourInMonth Whether or not there are multiple children turning 4 next month
+     * @return The build email payload.
+     */
+    public static EmailMessagePayload buildChildTurnsFourNotificationEmailPayload(PaymentCycle paymentCycle,
+                                                                                  PaymentCycleVoucherEntitlement entitlementNextMonth,
+                                                                                  boolean multipleChildrenTurningFourInMonth) {
+        Map<String, Object> emailPersonalisation = createPaymentEmailPersonalisationMap(paymentCycle, entitlementNextMonth);
+        emailPersonalisation.put(EmailTemplateKey.MULTIPLE_CHILDREN.getTemplateKeyName(), multipleChildrenTurningFourInMonth);
+        return EmailMessagePayload.builder()
+                .claimId(paymentCycle.getClaim().getId())
+                .emailType(EmailType.CHILD_TURNS_FOUR)
+                .emailPersonalisation(emailPersonalisation)
+                .build();
+    }
+
+    private static Map<String, Object> createPaymentEmailPersonalisationMap(PaymentCycle paymentCycle, PaymentCycleVoucherEntitlement voucherEntitlement) {
```

Comment:
Why has `voucherEntitlement` been extracted to a separate argument? (maybe you discussed this morning??)

Reply:
Because we need to be emailing the about the entitlement that they will receive for their next payment, not their current payment (which is on the PaymentCycle).